src/chrome/client/chromeDebugAdapter/unconnectedCDA.ts

```diff
@@ -60,11 +60,22 @@ export class UnconnectedCDA implements IDebugAdapterState {
     private parseLoggingConfiguration(args: ILaunchRequestArgs | IAttachRequestArgs): ILoggingConfiguration {
         let traceValue: Logger.LogLevel;
         switch (args.trace) {
+            case true:
+            case 'true':
```

Comment:
Why string `"true"`?

Reply:
In case someone mistakenly sends "true" instead of true.

Reply:
Well they should not do that...

Reply:
Removing it.